wiki/Tournaments/CHT/1/en.md

```diff
@@ -0,0 +1,476 @@
+---
+tags:
+  - CHT
+---
+
+# Coffee Hispanic Tournament
+
+![CHT logo](img/logo.jpg)
+
+The **Coffee Hispanic Tournament** (***CHT***) was a hispanic single-elimination 2v2 osu! tournament hosted by ![][flag_AR] [Vaf](https://osu.ppy.sh/users/12589048), ![][flag_AR] [iNeuu\_Z](https://osu.ppy.sh/users/9112463), and ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811). This was the first installment of the Coffee Hispanic Tournament.
```

Comment:
unclear what the qualifier "hispanic" means here, were you required to speak spanish to enter? or restricted to some countries?

Reply:
I think host meant countries that speak spanish, General Rules no. 3 if you want to check

Reply:
could use some clarification here then, I think you should remove hispanic from first sentence (it's already in the title) and add one more sentence after that saying it's country restricted

Reply:
```suggestion
The **Coffee Hispanic Tournament** (***CHT***) was a country-restricted single-elimination 2v2 osu! tournament hosted by ![][flag_AR] [Vaf](https://osu.ppy.sh/users/12589048), ![][flag_AR] [iNeuu\_Z](https://osu.ppy.sh/users/9112463), and ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811). This was the first installment of the Coffee Hispanic Tournament.
```
Rather this?

Reply:
```suggestion
The **Coffee Hispanic Tournament** (***CHT***) was a single-elimination 2v2 osu! tournament hosted by ![][flag_AR] [Vaf](https://osu.ppy.sh/users/12589048), ![][flag_AR] [iNeuu\_Z](https://osu.ppy.sh/users/9112463), and ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811). The tournament was restricted to Spanish-speaking countries only. This was the first installment of the Coffee Hispanic Tournament.
```
or something like that